### PR TITLE
Aguspina session expires

### LINKED ISSUE #189: Session expires very often
At least once a day (and perhaps more too, I don't really know), when I enter the platform my session is expired and I have to "Login with GitHub" again.

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes #189

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -61,6 +61,11 @@ def passed_solutions
     solutions.where(status: Status::Passed.to_i)
   end
 
+  def create_remember_me_token!
+    self.remember_me_token ||= get_token
+    self.save!
```
Comment (flbulgarelli) [inline app/models/user.rb]: We should move this to WithToken mixin

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,13 +1,14 @@
 class SessionsController < ApplicationController
   def create
     user = User.omniauth(env['omniauth.auth'])
-    session[:user_id] = user.id
+    cookies.permanent.signed[:mumuki_remember_me_token] = user.remember_me_token
```
Comment (flbulgarelli) [inline app/controllers/sessions_controller.rb]: We should extract this `cookies.permanent.signed` to somewhere else.

**Comment by aguspina** [comment]
It's ok for me!!

**Comment by flbulgarelli** [comment]
But, is it ok to remember the user whouth asking her about that?

I think this feature should be only enabled on demand, with and standard remember me/stay signed in checkbox.

**Pull request merged**